account_financial_report_menu/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Account Financial Report Menu",
+    "summary": """
+        Change the menu position "Invoicing/Reports/OCA accounting reports" to "Invoicing/OCA accounting reports"
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "13.0.1.0.0",
+    "category": "Account",
+    "website": "https://github.com/solvosci/slv-account",
+    "depends": [
+        "account_financial_report",
```

Comment:
Please add to `oca_dependencies.txt` a reference to `account-financial-reporting`.